MovingBlocks/TerasologyLauncher
# fix(i18n): update Ukrainian locale strings

### Contains

A yearly PR bringing the Ukrainian localization up to date!
This one adds 4 new strings and upgrades several others to match the English bundle.

### How to test

Launch the launcher, switch the language to Ukrainian, then
- take a look in the options menu
- try and activate an error with source files to see if things look alright

### Outstanding before merging

N/A
